controllers/task.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2015-2016, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const express = require('express');
+const lighthouseLib = require('../lib/lighthouse');
+const router = express.Router(); // eslint-disable-line new-cap
+
+/**
+ * POST /task/lighthouse/:pwaid
+ *
+ * Fetch and store a lighthouse report for the pwaid
+ */
+router.post('/lighthouse/:pwaid', (req, res, next) => {
```

Comment:
@andreban I think the two options seems to be either OAuth2 or ReverseDns the source IP.